wflambda.go

```diff
@@ -0,0 +1,283 @@
+package wflambda
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/aws/aws-lambda-go/lambdacontext"
+	"github.com/rcrowley/go-metrics"
+	"github.com/wavefronthq/go-metrics-wavefront"
+	"log"
+	"os"
+	"reflect"
+	"strings"
+	"time"
+)
+
+type lambdaHandlerFunction func(context.Context, json.RawMessage) (interface{}, error)
+
+var (
+	server                  string
+	authToken               string
+	isReportStandardMetrics bool
+	csCounter               metrics.Counter
+	csEventCounter          metrics.Counter
+	invocationsCounter      metrics.Counter
+	invocationEventCounter  metrics.Counter
+	eCounter                metrics.Counter
+	eEventCounter           metrics.Counter
+	dGauge                  metrics.Gauge
+	isColdStart             = true
+)
+
+/* Returns the Wavefront Lambda wrapper. The wrapper collects aws lambda's
+   standard metrics and reports it directly to the specified wavefront url. It
+   requires the following Environmental variables to be set:
+   1.WAVEFRONT_URL : https://<INSTANCE>.wavefront.com
+   2.WAVEFRONT_API_TOKEN : Wavefront API token with Direct Data Ingestion permission
+   3.IS_REPORT_STANDARD_METRICS : Set to False to not report standard lambda
+                                 metrics directly to wavefront.
+*/
+func Wrapper(lambdaHandler interface{}) interface{} {
```

Comment:
Is it possible to make the wrapper just return `(interface{}, error)` instead of returning a lambdaErrorHandler? Or is that what aws recommends?